index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
```

Comment:
<title></title> 태그는 웹 페이지의 제목을 나타내는 태그인데, 검색 엔진 등에서 가장 크게 보여지는 텍스트이므로 페이지의 특성을 드러낼 수 있도록 수정하는 게 좋을 것 같아요!

```suggestion
    <title>도와줘 햄식!</title>
```